types/sindresorhus__df/sindresorhus__df-tests.ts

```diff
@@ -0,0 +1,37 @@
+import df from '@sindresorhus/df';
```

Comment:
Should be `import df = require('@sindresorhus/df')`, or could leave as-is and add `"esModuleInterop": true` to `tsconfig.json` compilerOptions.